core/tests/mixin.py

```diff
@@ -51,3 +60,22 @@ def setUp(self):
         )
         self.user.id = self.user.idUsuario
         self.client = self.get_client()
+
+
+MEDIA_ROOT = tempfile.mkdtemp()
+
+
+@override_settings(MEDIA_ROOT=MEDIA_ROOT)
+class APIImageTestMixin(APITestMixin, TestCase):
```

Comment:
```suggestion
class APIImageTestMixin(APITestMixin):
```
Acredito não ser necessário. Fica melhor sempre colocar no método de teste específico (pelo menos eu acho) 👨‍👩‍👧‍👦👨‍👩‍👦‍👦👨‍👩‍👧‍👧

Reply:
Até tentei, mas dá erro
![image](https://user-images.githubusercontent.com/54081877/162348847-8659ae87-167c-413d-a61c-2a617583a2a6.png)